plugins/react-native.js

```diff
@@ -7,6 +7,16 @@
  *   var Raven = require('raven-js');
  *   require('raven-js/plugins/react-native')(Raven);
  */
+
+var DEVICE_PATH_RE = /\/var\/mobile\/Containers\/Bundle\/Application\/[^\/]+\/[^\.]+\.app/;
+function normalizeUrl(url) {
+    "use strict";
+
+    return url
+        .replace(/file\:\/\//, '')
```

Comment:
Should qualify this with a `^` so it doesn't replace this randomly in the string.